Makefile

```diff
@@ -18,10 +18,9 @@ COVER_PROFILE = cover.out
 BIN_DIR := bin
 
 CRD_OPTIONS ?= "crd:trivialVersions=false,allowDangerousTypes=true,crdVersions=v1"
-CONTROLLER_TOOLS_VERSION=v0.4.1
-CONTROLLER_GEN := $(BIN_DIR)/controller-gen
-GOLANGCI_LINT := $(BIN_DIR)/golangci-lint
-KUSTOMIZE := $(BIN_DIR)/kustomize
+CONTROLLER_GEN ?= go run sigs.k8s.io/controller-tools/cmd/controller-gen
+GOLANGCI_LINT ?= GOLANGCI_LINT_CACHE=$(GOLANGCI_LINT_CACHE) go run github.com/golangci/golangci-lint/cmd/golangci-lint
+KUSTOMIZE ?= go run sigs.k8s.io/kustomize/kustomize/v3
```

Comment:
Do we still need to keep [tools/install_kustomize.sh](https://github.com/metal3-io/baremetal-operator/blob/master/tools/install_kustomize.sh) ?

Reply:
no, i'll remove, thanks